client/src/app/app.component.ts

```diff
@@ -1,10 +1,23 @@
 import { Component } from '@angular/core';
+import { Router } from '@angular/router';
+import { AuthService } from './auth/services/auth.service';
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
-  styleUrls: ['./app.component.sass']
+  styleUrls: ['./app.component.sass'],
 })
 export class AppComponent {
   title = 'bugetify';
+
+  constructor(private authService: AuthService, private router: Router) {}
+
+  get isLoggedIn() {
+    return this.authService.isLoggedIn();
+  }
+
+  logout() {
```

Comment:
please, add modifiers public/private/protected before every function and add return type of the function